build.sbt

```diff
@@ -496,18 +497,14 @@ lazy val tests: ProjectMatrix = (projectMatrix in file("tests"))
   )
   .dependsOn(core, files, circeJson, cats)
 
-val akkaHttpVanilla = taskKey[Unit]("akka-http-vanilla")
-val akkaHttpTapir = taskKey[Unit]("akka-http-tapir")
-val akkaHttpVanillaMulti = taskKey[Unit]("akka-http-vanilla-multi")
-val akkaHttpTapirMulti = taskKey[Unit]("akka-http-tapir-multi")
-val http4sVanilla = taskKey[Unit]("http4s-vanilla")
-val http4sTapir = taskKey[Unit]("http4s-tapir")
-val http4sVanillaMulti = taskKey[Unit]("http4s-vanilla-multi")
-val http4sTapirMulti = taskKey[Unit]("http4s-tapir-multi")
-def genPerfTestTask(servName: String, simName: String) = Def.taskDyn {
-  Def.task {
-    (Compile / runMain).toTask(s" sttp.tapir.perf.${servName}Server").value
-    (Gatling / testOnly).toTask(s" sttp.tapir.perf.${simName}Simulation").value
+val perfTestCommand = Command.args("perf", "<servName> <simName>") { (state, args) =>
+  args match {
+    case Seq(servName, simName) =>
+      System.setProperty("tapir.perf.serv-name", servName)
+      Command.process(s"perfTests/Gatling/testOnly sttp.tapir.perf.${simName}Simulation", state)
```

Comment:
using the `run` task (or is it a command?) w/ arguments doesn't work?

Reply:
then maybe we could have `(Compile / runMain).toTask(s"sttp.tapir.perf.Server").value` with dynamically-determined arguments? Then `Server` could pattern-match on the required server impl and run it

Reply:
We still need dynamic, which doesn't work with tasks. Additionally, running and stopping server in the simulation using `before` and `after` seems to give a better control over the server.

Reply:
ok, sounds reasonable - put it in a comment for posterity ;)